### PR TITLE
add new column and create new data source in new release

### PR DESCRIPTION
**Pull request by alrocar**
Creates a new Release 0.0.3 and deploys a new bq_pypi Data Source version with the `test` column.

The previous version without the `test` column still exists in the Release 0.0.2, so you can promote / rollback between Releases to activate one or the other.

Screenshot 2023-12-05 at 23 58 09

### CODE REVIEW

**Comment by alrocar** [comment]
@xavijam 

Some things to mention:

- There are previous PR doing an alter of the `bq_pypi.datasource`. Those PRs are equivalent to the previous use case.
- We need to `tb datasource rm` in the custom deployment because `--fork-downstream` is not working for BigQuery Data Sources (issue already reported). If the BQ datasource had dependencies this use case would be more difficult to do.
- After merging we need to run the `promote` job.

File: v3/add_column_BQ_ds/deploy/0.0.3/cd-deploy.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+tb release create --semver ${VERSION}
+tb --semver ${VERSION} datasource rm bq_pypi --yes
+tb --semver ${VERSION} deploy --fixtures --populate --wait
+tb release preview --semver ${VERSION}
```
Comment (albertojuan) [inline v3/add_column_BQ_ds/deploy/0.0.3/cd-deploy.sh:7]: I guess there will be another PR to backfill and sync the new bq_pypi Data Source in preview with the live one, right?

**Review by albertojuan — APPROVED** [review]
